### PR TITLE
Update sidebar styles

### PR DESCRIPTION
**Pull request by imransl**
## Description
Updated styles in the sidebars of Species page, Entity viewer and Genome browser based on the styling provided by Andrea in this XD: https://xd.adobe.com/view/c6128e60-17d3-4ca6-a9cd-03d52f51b7fd-9341/.

I have updated the relevant CSS modules and have also used `ExternalLink` for the annotation provider links in Species page sidebar.

## Related JIRA Issue(s)
[ENSWBSITES-2409](https://www.ebi.ac.uk/panda/jira/browse/ENSWBSITES-2409)

## Deployment URL(s)
http://update-sidebar-styles.review.ensembl.org

## Views affected
Species page, Entity viewer and Genome browser

### CODE REVIEW

File: src/content/app/genome-browser/components/track-panel/components/track-panel-list/track-panel-items/TrackPanelItem.module.css [hunk 1]
```diff
@@ -40,4 +40,5 @@
   display: flex;
   column-gap: 5px;
   padding: 4px 10px;
+  --pill-button-font-size: 11px;
```
Comment (azangru) [inline src/content/app/genome-browser/components/track-panel/components/track-panel-list/track-panel-items/TrackPanelItem.module.css:43]: Here and below. If you check stylelint warnings, doesn't it suggest putting variable declarations at the top within a block of rules?

File: src/content/app/entity-viewer/gene-view/components/gene-view-sidebar/overview/GeneOverview.module.css [hunk 2]
```diff
@@ -22,7 +22,7 @@
 
 .geneSymbol {
   font-weight: var(--font-weight-bold);
-  margin-right: 10px;
+  margin-right: 12px;
 }
```
Comment (azangru) [inline src/content/app/entity-viewer/gene-view/components/gene-view-sidebar/overview/GeneOverview.module.css:27]: looks like the left margin here is off:

image

XD: 12 + 12; us: 12 + 15.

Comment (azangru) [reply src/content/app/entity-viewer/gene-view/components/gene-view-sidebar/overview/GeneOverview.module.css:27]: Something-something line height. Notice how the downward strokes of letters in the biotype get cropped:

image

File: src/content/app/entity-viewer/variant-view/variant-view-sidebar/overview/VariantOverview.module.css [hunk 3]
```diff
@@ -35,11 +35,11 @@
...
   font-weight: var(--font-weight-light);
-  margin-left: 0.6rem;
+  margin-left: 12px;
 }
 
 .labelTextStrong {
-  font-size: 12px;
+  font-size: 13px;
   font-weight: var(--font-weight-bold);
 }
```
Comment (azangru) [inline src/content/app/entity-viewer/variant-view/variant-view-sidebar/overview/VariantOverview.module.css:45]: Here's what this whole review started with, isn't it? Compare this to the styles in the XD

image

File: src/content/app/genome-browser/components/drawer/drawer-views/variant-summary/variant-consequence/VariantConsequence.module.css [hunk 4]
```diff
@@ -1,4 +1,5 @@
 .variantConsequence {
+  font-size: 13px;
```
Comment (azangru) [inline src/content/app/genome-browser/components/drawer/drawer-views/variant-summary/variant-consequence/VariantConsequence.module.css:2]: Why is this necessary?

File: src/content/app/entity-viewer/variant-view/variant-view-sidebar/overview/VariantOverview.module.css [hunk 5]
```diff
@@ -35,11 +35,11 @@
 .variantType {
   font-size: 12px;
   font-weight: var(--font-weight-light);
-  margin-left: 0.6rem;
+  margin-left: 12px;
 }
 
 .labelTextStrong {
-  font-size: 12px;
+  font-size: 13px;
```
Comment (azangru) [inline src/content/app/entity-viewer/variant-view/variant-view-sidebar/overview/VariantOverview.module.css:42]: this is the default body font size; shouldn't need to specify it unless you are overriding some other styles

File: src/content/app/genome-browser/components/track-panel/components/track-panel-list/track-panel-items/TrackPanelItem.module.css [hunk 6]
```diff
@@ -4,16 +4,16 @@
 }
 
 .label > *:not(:last-child, .itemsCount) {
-  margin-right: 10px;
+  margin-right: 12px;
 }
 
 .labelText {
-  font-size: 12px;
+  font-size: 13px;
```
Comment (azangru) [inline src/content/app/genome-browser/components/track-panel/components/track-panel-list/track-panel-items/TrackPanelItem.module.css:11]: Wherever you are specifying font-size: 13; you probably don't need to.

File: src/shared/components/copy/Copy.module.css [hunk 7]
```diff
@@ -4,6 +4,7 @@
   padding: 0 14px;
   align-items: center;
   justify-content: center;
+  font-size: var(--var-copy-font-size);
```
Comment (azangru) [inline src/shared/components/copy/Copy.module.css:7]: 1) Why use `var` as part of the variable name?
2) If you want to make font size of this component customisable, provide the default value (`font-size: var(--copy-button-font-size, 12px);`. Confirm with Andrea whether she plans to always use 12px for this element.
3) Unless parent component needs to override this style rule, there's no need to define it in the parent component (see your changes in VariantVCF.module.css)

File: src/content/app/genome-browser/components/track-panel/components/track-panel-list/track-panel-items/TrackPanelItem.module.css [hunk 8]
```diff
@@ -4,16 +4,15 @@
...
 }
 
 .labelText {
-  font-size: 12px;
   line-height: 1;
 }
 
 .labelTextStrong {
-  font-size: 12px;
+  font-size: 13px;
```
Comment (azangru) [inline src/content/app/genome-browser/components/track-panel/components/track-panel-list/track-panel-items/TrackPanelItem.module.css:15]: Are you sure this font-size rule is necessary? I unticked it in dev tools, and didn't see any change in the track panel.

Comment (imransl) [reply src/content/app/genome-browser/components/track-panel/components/track-panel-list/track-panel-items/TrackPanelItem.module.css:15]: No we don't need this. I forgot to remove this one. Thanks for spotting it.
